blocks/identity-block/components/HeadlinedSubmitForm/styles.scss

```diff
@@ -1,36 +1,39 @@
 .xpmedia-form-headline {
-	border-bottom: 1px solid $border-rule-color;
-	padding-bottom: map-get($spacers, "sm");
-	margin-bottom: map-get($spacers, "sm");
+	border-bottom: 1px solid #dadada;
+	font-size: 1.5rem;
+	font-weight: 700;
+	line-height: 2rem;
+	padding-bottom: 1rem;
+	margin-bottom: 1rem;
```

Comment:
```suggestion
	border-bottom: 1px solid #dadada;
	font-size: var(--global-font-size-9);
	font-weight: var(--global-font-weight-7);
	line-height: var(--global-spacing-6);
	padding-bottom: var(--global-spacing-4);
	margin-bottom: var(--global-spacing-4);
```

Reply:
ah ok. Will update all the styles with tokens